src/components/Tabs/Tabs.stories.tsx

```diff
@@ -0,0 +1,182 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import React, { useState } from "react";
+import { Meta, Story } from "@storybook/react";
+import { Tabs, TabSize, TabsPaddingX, TabsProps } from "./Tabs";
+import { Button } from "@components/Button";
+import { Text } from "../../typography/Text";
+import { Divider } from "@components/Divider";
+import { TabItem } from "@components/Tabs/TabItem";
+import { IconIcons, IconSize } from "@foundation/Icon";
+import { BadgeStyle } from "@components/Badge";
+
+const data = [
```

Comment:
```suggestion
const data: TabItemProps[] = [
```
This will require to rename `content` to `children`
Same can be done on the array in `Tabs.stories.tsx`